cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) idoneam (2016-2020)
+#
+# This file is part of Canary
+#
+# Canary is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Canary is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Canary. If not, see <https://www.gnu.org/licenses/>.
+
+# discord-py requirements
+import discord
+from discord.ext import commands
+import asyncio
+
+# Other utilities
+import random
+import sqlite3
+from .utils.paginator import Pages
+import time
+
+
+class Customreactions(commands.Cog):
```

Comment:
I did this because I didn't want to put a capital letter in the filename, but now jidicula told me the class name can have different caps than the filename so i'll fix this